src/hooks/useAnalyticsUI.ts

```
@@ -0,0 +1,326 @@
+        button_id: buttonId,
+        button_type: buttonType,
+        ...context,
+        timestamp: Date.now(),
+      })
+    } catch (error) {
+      console.error('Analytics: Failed to track button click:', error)
+    }
+  }, [enabled, trackButtons])
+
+  // Track navigation interactions
+  const trackNavigation = useCallback(async (fromPage: string, toPage: string, method: 'link' | 'button' | 'back' | 'forward', context?: Record<string, any>) => {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'trackNavigation'. Consider to delete it or rename it.

'trackNavigation' is defined here:


(lint/suspicious/noRedeclare)